src/whylogs/app/logger.py

```diff
@@ -62,6 +64,7 @@ def __init__(self,
                  cache_size: int = 1,
                  segments: Optional[Union[List[Segment], List[str]]] = None,
                  profile_full_dataset: bool = False,
+                 constraints: DatasetConstraints = None,
```

Comment:
need to add docstring above

Reply:
added `constraints: static assertions to be applied to streams or summaries.`